programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+    n = String(n).split(``);
+    let answer = 0;
+    for(let i = 0; i <= n.length-1; i++) {
```

Comment:
메모리 구조 때문에<= 보다는 <가 더욱 좋습니다.
그런데 값을  - , + 하면서 까지 바꿀 필요는 없는데
여기서는 - 하면서 <= 까지 사용하게 되니 -1을 하지않고 < 를 사용하는것이 더욱 좋아보입니다.
```suggestion
    for(let i = 0; i < n.length; i++) {
```